openapi/listings/listings.yaml

```diff
@@ -50,8 +50,8 @@ paths:
         - schema:
             type: string
           in: query
-          name: 'filter[cursor]'
-          description: If there are more than 200 listing sources they will be paginated into pages of 200. This is the cursor to the current page in the results
+          name: 'page[cursor]'
```

Comment:
You should add `page[limit]` as well or at the very least document your intention of returning the first 200 all the time.